ITensorGPU/test/runtests.jl

```diff
@@ -1,6 +1,22 @@
-using ITensorGPU, Test, CUDA
+if VERSION < v"1.8"
+  itensorgpu_path = joinpath(pwd(), "..")
+  cp(
+    joinpath(pwd(), "..", "LocalPreferences.toml"),
+    joinpath("$(LOAD_PATH[2])", "JuliaLocalPreferences.toml"),
+  )
+  #old_path = pop!(LOAD_PATH)
+  #push!(LOAD_PATH, itensorgpu_path)
+  #push!(LOAD_PATH, old_path)
+end
+
+using CUDA
+println("Running ITensorGPU tests with a runtime CUDA version: $(CUDA.runtime_version())")
+
+using ITensorGPU, Test
 
 CUDA.allowscalar(false)
+using Pkg
+Pkg.add("Combinatorics")
```

Comment:
We shouldn't add packages like this inside the tests. Either they will get installed automatically with `Pkg.test`, or you should activate and instantiate the test environment to install the dependencies.